src/main.rs

```diff
@@ -93,7 +99,8 @@ fn main() {
 
 #[allow(deprecated)]
 fn err_cause(err: &dyn std::error::Error, max: usize) {
-    if let Some(reason) = err.cause() { // we use cause(), not source()
+    if let Some(reason) = err.cause() {
+        // we use cause(), not source()
```

Comment:
Please don't use rustfmt. The formatting in this crate is deliberate, and `rustfmt` makes it worse

Reply:
i see, will revert